requirements.txt

```diff
@@ -10,11 +11,12 @@ boto3==1.9.199
 owslib~=0.18.0
 requests_file~=1.4.3
 natsort~=6.0.0
-pillow~=6.1.0
-pyproj==1.9.6
-psycopg2
+pillow
+pyproj
```

Comment:
I unpinned these, along with the `Werkzeug` dependency because they were very old and cause `pip install` failures (at least on macOS). I suspect others pinned items are old as well, but of course, feel free to ignore the unpinning or repin to recent versions.

Reply:
This should be fine. The code will be published to our dit server once we merge to the `develop` branch at which point we can check for any compatibility issues.